modules/configuration/base_config.lua

```diff
@@ -46,6 +46,9 @@ config.webengineUniqueId = "5lNGKj5n63Hf8QjZPzS7X5yjZWYD0nu6IhKtyQOFRuUZkbY8KOdF
 
 --- Logs and Reports
 --
+
+-- Flag which defines the logger name (BOOST, LOG4CXX)
+config.loggerName = "BOOST"
```

Comment:
I think we should leave the default as `LOG4CXX`

Reply:
Agreed, changed in https://github.com/smartdevicelink/sdl_atf/pull/216/commits/314ae47670017992a0038790320432e3649187d5